docs/conf.py

```diff
@@ -0,0 +1,83 @@
+# encoding: utf-8
+
+"""pypcap documentation build configuration file"""
+
+from __future__ import unicode_literals
+
+import os
+import re
+import sys
+import types
+import mock
+
+
+def get_version(filename):
+    # Hack this for right now since pypcap doesn't have standard module setup
+    return '1.1.5'
```

Comment:
Ugh.

Yeah we should come up with some better system for managing versioning, maybe one way to go is to create a simple shim around the c extension that we call _pypcap and have a pure python wrapper around it that is the actual pypcap.

Another way to do it would be to put version information in a variable in our setup.py and import it from there.